nemo/collections/asr/parts/utils/decoder_timestamps_utils.py

```diff
@@ -348,6 +348,17 @@ def set_asr_model(self):
             self.asr_batch_size = if_none_get_default(self.params['asr_batch_size'], 4)
             self.model_stride_in_secs = 0.02
 
+        elif 'fastconformer' in self.ASR_model_name.lower():
+            self.run_ASR = self.run_ASR_BPE_CTC
+            self.encdec_class = EncDecCTCModelBPE
+            self.decoder_delay_in_sec = if_none_get_default(self.params['decoder_delay_in_sec'], 0.08)
+            self.word_ts_anchor_offset = if_none_get_default(self.params['word_ts_anchor_offset'], 0.12)
+            self.asr_batch_size = if_none_get_default(self.params['asr_batch_size'], 16)
+            self.model_stride_in_secs = 0.08
+            # fastConformer requires buffered inference and the parameters for buffered processing.
+            self.chunk_len_in_sec = 5
+            self.total_buffer_in_secs = 25
```

Comment:
Thanks for the PR, have you experimented with these values?

Reply:
No, I didn't change these values, I only have experimented with the stride seconds